destTransformer.js

```diff
@@ -8,7 +8,9 @@ const { testRouter } = require("./testRouter");
 const cluster = require("./util/cluster");
 const { addPrometheusMiddleware } = require("./middleware");
 
-const clusterEnabled = process.env.CLUSTER_ENABLED !== "false";
+const clusterEnabled = process.env.CLUSTER_MODE
+  ? process.env.CLUSTER_MODE.toLowerCase() === "true"
+  : false;
```

Comment:
Let's use simpler syntax,

```suggestion
const clusterEnabled = process.env.CLUSTER_MODE?.toLowerCase() === "true";
```

Reply:
Right, transformer is in node 14 now. Added change